### ISSUE TITLE
Fix Cassandra storage to support latest Cassandra

### ISSUE THREAD

**Issue by jsccast** [Comcast/rulio#41]
Currently, you'll get "unconfigured table locstate" when running against the latest version (3.11.2) of Cassandra.

See #40.

**Comment by rakesh-roshan** [comment]
I am also getting the same error. You are saying this is the problem with the latest cassandra version. So did any old version is good?

**Comment by rakesh-roshan** [comment]
There is a problem in the locstate table definition. Nothing to do with cassandra version.  I have created a pull request to fix that. 

https://github.com/Comcast/rulio/pull/42

**Comment by maitreytalware** [comment]
I am still getting the same error "unconfigured table locstate", so I manually create the table in Cassandra, now the engine works fine.
But when I restart the engine, the engine is not able to search the previously stored facts (they are visible in Cassandra console)

**Comment by rakesh-roshan** [comment]
Did you pull the latest code? It is working fine for me.

**Comment by maitreytalware** [comment]
Yes, I did. Can you please restart the engine and search previously stored facts using API

**Comment by rakesh-roshan** [comment]
I did and it is working fine. In query are you providing correct location value?
Can you please dump the location table rows here and query which you are submitting?
